test/statetest/statetest_loader.cpp

```diff
@@ -221,6 +221,11 @@ state::Transaction from_json<state::Transaction>(const json::json& j)
     if (const auto ac_it = j.find("accessList"); ac_it != j.end())
         o.access_list = from_json<state::AccessList>(*ac_it);
 
+    o.nonce = from_json<uint64_t>(j.at("nonce"));
+    o.r = from_json<intx::uint256>(j.at("r"));
+    o.s = from_json<intx::uint256>(j.at("s"));
+    o.v = static_cast<uint8_t>(from_json<uint64_t>(j.at("v")));  // FIXME: remove static_cast
```

Comment:
Why not?
```suggestion
    o.v = from_json<uint8_t>(j.at("v"));
```

Reply:
Done. I added `from_json` function specialisation for this type.